src/main/java/org/isf/therapy/gui/TherapyEdit.java

```diff
@@ -92,6 +92,13 @@
 
 public class TherapyEdit extends ModalJFrame implements VisitListener {
 
+	private static final long serialVersionUID = 1L;
```

Comment:
If we are improving this file then there are also 6 occurrences of `+ " " + patient.getName()` in messages that should be format type messages and often need a final `?` or `.`.  Not sure whether you want to make that change too or I can do it after this PR is committed.

Reply:
I think in another PR can be addressed properly (don't know why it is still there)

Reply:
(I mean, I remember you did a complete pass about these kind of changes)

Reply:
anyway, let me do it now

Reply:
Done